components/LabelChip/index.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
```

Comment:
@alexfauquette [question] pk a-t-on besoin de "use client"?

Reply:
Because of the `styled()`. Did not went too deep about that because those chips will have a callback `onDelete` so they will be used in client components